backend/Auth/ResetPassword.ts

```diff
@@ -115,6 +119,63 @@ router.get<Params, ResponseBody, QueryParams>(
   }
 );
 
+router.put(
+  "/api/passwords",
+  body("password").isStrongPassword(),
+  body("token").exists(),
+  async (req, res) => {
+    const { password, token } = req.body;
+
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      if (token) {
```

Comment:
We don't need this if statement - the `errors.toArray()` method will already give us the error messages that are relevant to this request.

```ts
if (!errors.isEmpty()) {
  return invalidRequest(res, errors.toArray())
}
```